src/pks/basic/ParabolicPDE_PredictorCorrector.hh

```diff
@@ -0,0 +1,76 @@
+/*
+  ATS is released under the three-clause BSD License.
+  The terms of use and "as is" disclaimer for this license are
+  provided in the top-level COPYRIGHT file.
+
+  Authors: Ethan Coon (ecoon@lanl.gov)
+*/
+//! A parabolic PDE solved using a predictor-corrector scheme.
+
+/*!
+
+This is the canonical nonlinear parabolic PDE, in mixed form for
+predictor-corrector methods.
+
+.. math::
+    \frac{\partial \Psi(u) }{\partial t} - \nabla \cdot K(u) \nabla \Phi(u) = Q(u,x,t)
+
+where:
+
+- :math:`u` the primary variable, key_
+- :math:`\Psi` the conserved quantity, conserved_quantity_key_
+- :math:`\Phi` the potential field, potential_key_
+- :math:`K` the diffusion coefficient
+- :math:`Q` any source term, source_key_
+
+Any of these may be a function of the primary variable :math:`u`.
+
+Note that frequently \Phi(u) = u (e.g. the potential field is the primary
+variable) but not always -- specifically for surface water where the potential
+field is :math:`\delta(p) + z` for primary variable pressure.
+
+This simply uses the Explicit scheme as a predictor for the implicit scheme.
```

Comment:
What is the value of this scheme? Is it conditionally stable, 2nd-order, etc? In general, should we place time integration schemes in its designated directory, since could be so many of them.